src/HH4b/run_utils.py

```diff
@@ -188,6 +187,13 @@ def parse_common_args(parser):
         default="2022",
         choices=["2018", "2022", "2022EE", "2023", "2023BPix"],
     )
+    parser.add_argument(
+        "--pnet-txbb",
+        type=str,
+        required=True,
+        choices=["legacy", "v12", "part"],
```

Comment:
similarly, maybe the options should be `pnet-legacy`, `pnet-v12`, `glopart-v1` ? but i understand if that causes other headaches in the code